src/Files.App/Actions/Content/Install/InstallFontAction.cs

```diff
@@ -8,7 +8,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace Files.App.Actions
+namespace Files.App.Actions.Content.Install
```

Comment:
I'm pretty sure that the namespaces for the others actions is just `Files.App.Actions`.